components/admin_console/brand_image_setting.jsx

```diff
@@ -214,3 +203,45 @@ export default class BrandImageSetting extends React.PureComponent {
         );
     }
 }
+
+function UploadButton(props) {
```

Comment:
I think this should be moved into a separate file and made into a PureComponent. We have an ESLint rule against multiple components in a single file that this is bypassing

Reply:
I was thinking earlier about that too but this is specific to this component only.